editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -259,9 +258,9 @@ export default function Component(props: Props) {
                   drawGeojsonData={JSON.stringify(boundary)}
                   drawGeojsonDataBuffer={10}
                   clipGeojsonData={
-                    addressPoint &&
+                    boundary &&
                     JSON.stringify(
-                      buffer(addressPoint, bufferInMeters, { units: "meters" }),
+                      buffer(boundary, bufferInMeters, { units: "meters" }),
```

Comment:
great ! should we still use an `addressPoint` fallback here though for instances where we haven't been able to fetch a title boundary? (we know it's a sometimes spotty dataset!)